app/controllers/admin/application_controller.rb

```diff
@@ -1,8 +1,24 @@
 module Admin
   class ApplicationController < ApplicationController
     before_action :authorize_admin
+    before_action :assign_help_url
     after_action :verify_authorized
 
+    HELP_URLS = {
```

Comment:
**(comment, non-blocking):~** This seems like it would be beneficial to put into a constants file

Reply:
My feeling that it was simplest to have it near where we need it for now and as the feature expands we could move it.